misaligned.js

```diff
@@ -1,16 +1,17 @@
-const {expect} = require('chai')
-
+let color_map = []
+function getcolor_map(){

```

Comment:
Variables that are exported directly only holds initial value. 
`color_map` will always hold an empty array since it was intialised as an empty array. And `color_map` is getting updated later in the function `print_color_map`.
 So I through of using a function that returns updated value whenever it is called. 
And also for the test I only needed the value of  the variable `color_map` not the entire variable. So using a `getcolor_map` getter function will only allow me to get the value of the variable `color_map`